### ISSUE TITLE
The testing on agents-playground.livekit.io is not functioning properly

### ISSUE THREAD

**Issue by makeryangcom** [livekit/agents#646]
When using LiveKit Cloud on agents-playground.livekit.io, it keeps loading indefinitely, making it unusable and impossible to test.

![20240813162552](https://github.com/user-attachments/assets/5b56ca5f-8c0e-4909-b6cf-0cde2bb1e669)

**Comment by robert-sarosi** [comment]
same here for the agent. having the actual livekit cloud url, key and secret, but is still unable to connect.

<img width="1207" alt="Screenshot 2024-08-23 at 00 07 13" src="https://github.com/user-attachments/assets/6956952f-9489-44c1-98ef-9cdac3781369">

**Comment by keepingitneil** [comment]
This looks like an issue on your agent. It's likely that it doesn't have the `LIVKIT_URL` `LIVEKIT_API_KEY` and `LIVEKIT_API_SECRET` environment variables set properly

**Comment by makeryangcom** [comment]
Do you also need to set up your own agent when using cloud services? If so, where do I need to configure it?

**Comment by ryanleecode** [comment]
it works like 1/100 of the time. super frustrating

**Comment by ryanleecode** [comment]
https://github.com/livekit/agents/issues/717

**Comment by sjportalatin** [comment]
@MatheusRDG For me this seemed to be because I had included `agent_name` in the WorkerOptions. This enables explicit dispatch, which prevents it from automatically joining the playground.

```
agent_name: str = ""
"""Set agent_name to enable explicit dispatch. When explicit dispatch is enabled, jobs will
 not be dispatched to rooms automatically. Instead, you can either specify the agent(s) to
 be dispatched in the end-user's token, or use the AgentDispatch.createDispatch API"""  # noqa: E501
```

**Comment by MatheusRDG** [comment]
@sjportalatin thanks, I think in my case is just some small structure issue of the agent, already working.

**Comment by lodwf81** [comment]
> Do you also need to set up your own agent when using cloud services? If so, where do I need to configure it?
> 
> The agent is continuously loading, and both the microphone and camera permissions have been granted.
> 
> ![image](https://github.com/user-attachments/assets/eec04e62-8388-4d57-9623-476c260bced7)

I get the same is there any fix?

**Comment by pax-k** [comment]
same for me.

Tried running https://github.com/livekit-examples/agent-starter-node locally with `pnpm dev`

**Comment by xuguozhi** [comment]
> [@sjportalatin](https://github.com/sjportalatin) thanks, I think in my case is just some small structure issue of the agent, already working.

same issue, how did you fix that?

**Comment by makeryangcom** [comment]
ou need to run the local agent first

**Comment by xuguozhi** [comment]
> ou need to run the local agent first

yes, I deployed local agents and start playground with env LIVEKIT_API_KEY LIVEKIT_API_SECRET LIVEKIT_URL and fill the agent name. Then all is done.

**Comment by Deep-404** [comment]
Any solution guys?
